### PR TITLE
Eng 1720

### PR DESCRIPTION
**Pull request by ChrisSchreiber**
removes unnecessary helm providers and updates helm provider to 1.1.0

### CODE REVIEW

File: stacks/product-jenkins/main.tf [hunk 1]
```diff
@@ -27,7 +27,7 @@ provider "kubernetes" {
 
 provider "helm" {
```
Comment (cplee) [inline stacks/product-jenkins/main.tf]: do we still need all 3 helm providers?

File: stacks/product-local/main.tf [hunk 2]
```diff
@@ -35,7 +35,7 @@ provider "kubernetes" {
 }
 
 provider "helm" {
```
Comment (cplee) [inline stacks/product-local/main.tf]: do we still need all 3 helm providers?

File: stacks/product-local/main.tf [hunk 3]
```diff
@@ -9,7 +9,7 @@ provider "kubernetes" {
...
-  version         = "0.10.4"
+  version         = "1.1.0"
   alias           = "toolchain"
   namespace       = module.product_jenkins.toolchain_namespace
   tiller_image    = "gcr.io/kubernetes-helm/tiller:v2.15.1"
```
Comment (cplee) [inline stacks/product-local/main.tf]: remove tiller_image

File: stacks/product-local/main.tf (hunk 2 above)
Comment (ChrisSchreiber) [reply stacks/product-local/main.tf]: Not the way we are using the products now. If we wanted to be able to have the products deploy to other clusters we would but we should have to add a bunch of other stuff anyway.

Comment (cplee) [reply stacks/product-local/main.tf]: gotcha.  i'm fine keeping them around then

**Pull request merged**